.travis.yml

```diff
@@ -1,14 +1,11 @@
-language: node_js
-node_js:
-  - '8'
-cache:
-  directories:
-    - node_modules
-before_deploy:
-  - openssl aes-256-cbc -K $encrypted_5a663823105c_key -iv $encrypted_5a663823105c_iv -in ./scripts/id_rsa.enc -out /tmp/deploy_rsa -d
-  - eval "$(ssh-agent -s)"
-  - chmod 600 /tmp/deploy_rsa
-  - ssh-add /tmp/deploy_rsa
+node-preset: &node # Used internally to share configuration, ignored by travis.
+  language: node_js
+  node_js:
+    - "8"
+  cache:
+    yarn: true
```

Comment:
We removed the yarn cache in https://github.com/cozy/cozy-banks/pull/1081/files to get better perfs in the travis build. Do we need it?